ote/src/cljs/ote/app/controller/route/route_list.cljs

```diff
@@ -46,8 +72,48 @@
 
   LoadRoutesResponse
   (process-event [{response :response} app]
-    (assoc app :route-list response
-               :routes-vector (get-routes-for-operator app response)))
+    (-> app
+        (handle-routes-response response)))
+
+  LinkInterfaceResponse
+  (process-event [{is-linked? :is-linked?
+                   response :response} app]
+    (let [current-operator (get app :transport-operator)
```

Comment:
Not a big deal, but this could be destructured just like `response` and `is-linked`

Reply:
Could be destructed differently but to make code more readable I added it to let. And named it to "current" because it is not clear that selected operator is in app-state always in :transport-operator key.